projects/plugins/jetpack/extensions/blocks/button/button.php

```diff
@@ -109,6 +109,7 @@ function get_button_classes( $attributes ) {
 
 	if ( $has_font_size ) {
 		$classes[] = 'has-' . $attributes['fontSize'] . '-font-size';
+		$classes[] = 'has-custom-font-size';
```

Comment:
I am simultaneously sure this pr worked before this change, but also know that it couldn't have.

The jetpack/button will only save the `has-custom-font-size` class added in save.js if the `saveInPostContent` attribute is true. That attribute is never true for payment-buttons and so it relies upon this php code instead.

Reply:
> I am simultaneously sure this pr worked before this change, but also know that it couldn't have.

The Schrödinger's class!